src/format.js

```diff
@@ -84,160 +85,161 @@ export default function date(value, format, utc = false) {
     },
 
     // Month
-    F() {
+    F(): string {
       // Full month name; January...December
       return txtWords[6 + f.n()];
     },
-    m() {
+    m(): string {
       // Month w/leading 0; 01...12
       return zeroPad(f.n(), 2);
     },
-    M() {
+    M(): string {
       // Shorthand month name; Jan...Dec
       return f.F()
         .slice(0, 3);
     },
-    n() {
+    n(): number {
       // Month; 1...12
       if (utc) return jsdate.getMonth() + 1;
       return jsdate.getUTCMonth() + 1;
     },
-    t() {
+    t(): number {
       // Days in month; 28...31
       if (utc) return (new Date(f.Y(), f.n(), 0)).getUTCDate();
       return (new Date(f.Y(), f.n(), 0)).getDate();
     },
 
     // Year
-    L() {
+    L(): number {
       // Is leap year?; 0 or 1
-      const j = f.Y();
-      return j % 4 === 0 & j % 100 !== 0 | j % 400 === 0;
+      const j: number = f.Y();
+      return (j % 4 === 0 && j % 100 !== 0 || j % 400 === 0) ? 1 : 0;
```

Comment:
I had this error:
![image](https://user-images.githubusercontent.com/25547238/27013951-96416a1c-4ef6-11e7-87b8-2ef53e4cbbe5.png)
and i noticed that below  is using `? 1 : 0`,
so, i decided that it shoud be the same, if I replace bitwise `& |` to ternary statement that returns the same.
Or it doesnt returns the same?